### DISCUSSION TITLE
Native features and notifications on Android

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#1708] [Show and tell]
First of all, you're doing an amazing job!

I'm testing on Android, would like to know when access to native features and notifications will be available?

**Comment by FeodorFitsner** [comment]
Could you please elaborate what particular native Android features are you interested in? There is an issue for Notifications though: https://github.com/flet-dev/flet/issues/516

**Reply by nascin** [reply]
The native features would be the GPS and even if the cell phone was blocked, the application would still run, would that be possible?

**Reply by FeodorFitsner** [reply]
There is an issue for GPS support: https://github.com/flet-dev/flet/issues/66

You mean "cell phone was **locked**"?

**Reply by nascin** [reply]
that's right, it would be in the case of a running app, where it would be running, even with the screen locked.

**Reply by FeodorFitsner** [reply]
It's still to be investigated. Per my observation the app is suspended 30 seconds after lock. I would imagine some cloud service running your app logic and sending push notifications.

**Comment by Ayberk-Irmak** [comment]
Hey everyone! 👋

I finally found a solution for sending notifications in Flet mobile applications using Pyjnius!
Since I know that many of you have been struggling with this issue, I decided to make a video tutorial explaining the entire process.
Check it out, and let me know what you think!

🔗 Video Link: [How to Send Notifications in Flet using Pyjnius](https://www.youtube.com/watch?v=kXWKaylIhCg)

Hope it helps! 😊

**Reply by tamalmaity-dev** [reply]
Hey, Can you make for bluetooth in Android ...like bluetooth scan, and connect and receive data from another device ... here bluetooth classic and BLE support for Android..... I make this using Pyjinius ...but here only show get the paired devices ... but when tap to connect then not able to connect and only show the paired devices ..but I want show the available searchable devices ...

here I make like this ...here is my url: **[Bluetooth calling function using Pyjinius!](https://github.com/flet-dev/flet/discussions/5184)**

Please help me ... best regards
